assets/css/admin-custom.css

```diff
@@ -109,6 +109,15 @@ div.question_boolean_fields { margin-bottom: 10px; }
     padding-left: 2em !important;
 }
 
+.post-type-course .wp-list-table .button-link {
```

Comment:
It seems a bit dangerous to piggyback on the `button-link` class since WordPress uses this for its own styling.

Reply:
Yeah, I figured that out eventually. 😄 But I've updated the class name in the next PR here #4666.